[CI] :qa:mixed-cluster indices.stats/10_index/Index - all Failure

There have been a number of failures of this test on the master branch

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+bwc-tests/1262/console
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+intake/2339/console
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+default-distro-bwc-tests/180/console

In all cases the error is 
```
java.lang.AssertionError: Failure at [indices.stats/10_index:50]: field [indices.test1.uuid] doesn't have a true value
Expected: not null
     but: was null
```

The failure reproduces if you massage the reproduce line a little by changing `:qa:mixed-cluster:v6.4.0-SNAPSHOT#mixedClusterTestRunner` to `:qa:mixed-cluster:v6.4.0-SNAPSHOT#bwcTest` 
```
./gradlew :qa:mixed-cluster:v6.4.0-SNAPSHOT#bwcTest   -Dtests.seed=C06428A7AE0846BD   -Dtests.class=org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT   -Dtests.method="test {p0=indices.stats/10_index/Index - all}"   -Dtests.security.manager=true   -Dtests.locale=he-IL   -Dtests.timezone=NZ
```

The test has been muted in https://github.com/elastic/elasticsearch/commit/b655c11dbee65d5331819f04a15ace80b639cfbd

#31871 looks the likely cause

--- a/qa/mixed-cluster/build.gradle
+++ b/qa/mixed-cluster/build.gradle
@@ -57,7 +57,6 @@ for (Version version : bwcVersions.wireCompatible) {
   tasks.getByName("${baseName}#mixedClusterTestRunner").configure {
     /* To support taking index snapshots, we have to set path.repo setting */
     systemProperty 'tests.path.repo', new File(buildDir, "cluster/shared/repo")
-    systemProperty 'tests.rest.blacklist', ['indices.stats/10_index/Index - all'].join(',')
   }
 }
 

--- a/rest-api-spec/src/main/resources/rest-api-spec/test/indices.stats/10_index.yml
+++ b/rest-api-spec/src/main/resources/rest-api-spec/test/indices.stats/10_index.yml
@@ -41,6 +41,10 @@ setup:
 
 ---
 "Index - all":
+  - skip:
+      version: " - 6.3.99"
+      reason: "uuid is only available from 6.4.0 on"
+
   - do:
       indices.stats: { index: _all }
 

